athletics.lic

```diff
@@ -2,22 +2,26 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#athletics
```

Comment:
Yea I would like to see crossing-trainer use the rope. It might make sense to get the logic in athletics working and then have crossing-training call the athletics script if the rope is being used.